### PR TITLE
Updating base page elements with i18n and trans tag

### PR DESCRIPTION
**Pull request by nicklee**

### CODE REVIEW

File: donate/templates/pages/base_page.html [hunk 1]
```diff
@@ -61,22 +61,22 @@
         <div class="footer__nav footer-links">
             <ul class="footer-links__container">
                 <li class="footer-links__item">
-                    <a class="link link--dark-bg" href="https://twitter.com/mozilla" target="_blank" rel="noopener">Twitter</a>
+                    <a class="link link--dark-bg" href="https://twitter.com/mozilla" target="_blank" rel="noopener">{% trans "Twitter" %},</a>
```
Comment (solarissmoke) [inline donate/templates/pages/base_page.html]: Looks like you've accidentally added a comma at the end here.

File: donate/templates/pages/base_page.html [hunk 2]
```diff
@@ -61,22 +61,22 @@
...
                 </li>
                 <li class="footer-links__item">
-                    <a class="link link--dark-bg" href="https://www.facebook.com/mozilla" target="_blank" rel="noopener">Facebook</a>
+                    <a class="link link--dark-bg" href="https://www.facebook.com/mozilla" target="_blank" rel="noopener">{% trans "Facebook" %},</a>
```
Comment (solarissmoke) [inline donate/templates/pages/base_page.html]: Here too

File: donate/templates/pages/base_page.html (hunk 1 above)
Comment (nicklee) [reply donate/templates/pages/base_page.html]: oops, good spot!

File: donate/templates/pages/base_page.html (hunk 2 above)
Comment (nicklee) [reply donate/templates/pages/base_page.html]: Thanks!

**Pull request merged**